modules/common/createValidAzResourceName.bicep

```diff
@@ -24,6 +24,7 @@ param location string
   'uami' // User-assigned Managed Identity
   'dplscr' // Deployment Script
   'law' // Log Analytics Workspace
+  'avd' // Azure Virtual Desktop
```

Comment:
With this approach, we're naming all AVD infrastructure resources the same (host pool, workspace, and application group). We should think about this.